packages/desktop-client/src/components/budget/BudgetTotals.tsx

```diff
@@ -54,7 +56,7 @@ export const BudgetTotals = memo(function BudgetTotals({
           WebkitUserSelect: 'none',
         }}
       >
-        <View style={{ flexGrow: '1' }}>Category</View>
+        <View style={{ flexGrow: '1' }}>{t('Category')}</View>
```

Comment:
We should also translate the `aria-label` and the `text` strings below

Reply:
`aria-label={t('Menu')}` should work?

Reply:
Yep!

Reply:
And same for `text`

Reply:
mmm got `Type '{ t(: any): any; }' is not assignable to type 'string'.` while doing it for `text` :/

Reply:
in this case don't need `{ }` I guess.